maintenance_page/scripts/failover.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -eu
+
+NAMESPACE=$(jq -r '.namespace' terraform/aks/workspace_variables/${CONFIG}.tfvars.json)
```

Comment:
Should it be terraform/application/config/${CONFIG}.tfvars.json

Reply:
updated, and will fix in template